src/resources/application.rs

```diff
@@ -501,7 +501,7 @@ pub async fn list_apps(
     Ok(models)
 }
 
-async fn get_client(
+pub async fn get_client(
```

Comment:
Perhaps in a future refactor this could be re-homed outside of the application resource file

Reply:
I didn't quite want to introduce a `utils` module or crate, but it was tempting